build/release.sh

```diff
@@ -15,9 +15,7 @@ cmd() {
         return 0
     fi
 
-    "$@"
-
-    return 0
+    return "$@"
```

Comment:
Will the `$@` return anything? Should it be simply `"$@"` without any return statement?